oauthenticator.gemspec

```diff
@@ -27,8 +27,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'rack', '>= 1.4', '< 2.3'
   spec.add_runtime_dependency 'json', '~> 1.8'
-  spec.add_runtime_dependency 'faraday', '~> 0.9'
+  spec.add_runtime_dependency 'faraday', '~> 1.0.0'
```

Comment:
this should probably be 1.0, so that any of the minor releases would work (i.e. >= 1.0.0, < 2.0.0, vs >= 1.0.0, < 1.1.0 as you have it)